### PR TITLE
gpgv: add page

### PR DESCRIPTION
**Pull request by jameshibbard**
Hi,

Here's a PR adding an entry for the [gpgv](https://manpages.ubuntu.com/manpages/bionic/man1/gpgv.1.html) command.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/gpgv.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# gpgv
+
+> Verify OpenPGP signatures.
+> More information: <https://www.gnupg.org/documentation/manuals/gnupg/gpgv.html>.
+
+- Verify a signed file:
+
+`gpgv doc.txt`
```
Comment (zdroid) [inline pages/common/gpgv.md:8]:
```suggestion
`gpgv {{path/to/file}}`
```

File: pages/common/gpgv.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Verify a signed file using a detached signature:
+
+`gpgv sigfile.asc doc.txt`
```
Comment (zdroid) [inline pages/common/gpgv.md:12]:
```suggestion
`gpgv {{path/to/signature}} {{path/to/file}}`
```

File: pages/common/gpgv.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Add a file to the list of keyrings (a single exported key also accounts as a keyring):
+
+`gpgv --keyring ./alice.keyring sigfile.asc doc.txt`
```
Comment (zdroid) [inline pages/common/gpgv.md:16]:
```suggestion
`gpgv --keyring {{./alice.keyring}} {{path/to/file}} {{path/to/signature}}`
```

Comment (jameshibbard) [reply pages/common/gpgv.md]: Yup. Sorry. Done.

**Comment by sbrl** [comment]
Great to see more GPG commands documented in tldr! They are come of the most difficult to use, and can really benefit from tldr pages - thanks so much @jameshibbard :D

**Comment by jameshibbard** [comment]
My pleasure. I'm enjoying contributing :smile:

**Pull request merged**